ntbs-service/Pages/Shared/_NotificationBanner.cshtml

```diff
@@ -27,6 +27,10 @@
                 {
                     <div> #@Model.NotificationId </div>
                 }
+                @if(!@Model.IsEditable)
+                {
+                    <p> (not editable) </p>

```

Comment:
This is placeholder only, isn't? Otherwise it's not quite hitting the right tone, as it also means it's not viewable in full

Reply:
yes it's a placeholder! didn't wanna spend ages (well 10 minutes haha) looking for an image before we've decided what we actually want here. Could change the text to something else though?  "Unauthorized"? But that's not very clear either

Reply:
(limited access)?

Reply:
Did you just suggest this independently? Because that's what I went with ha.

Reply:
haha i did 😂
High ✋!